modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from '@emotion/styled-base';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas, {borderRadius} from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React from 'react';
+
+const ActivePage = styled('button')({
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  fontSize: 13,
```

Comment:
Is it possible to use canvas variables? I think even in labs, we should use canvas variables to prevent us from accidentally slipping from design tokens and introducing design drift